runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerConfig.java

```diff
@@ -48,4 +48,14 @@ default Color markerColor()
 	{
 		return Color.YELLOW;
 	}
+
+	@ConfigItem(
```

Comment:
I don't think this should be toggle

Reply:
From what others have said I think we should keep the toggle. Otherwise you could accidentally hit clear when you meant to mark. I agree it should be on by default however.